src/test/integration/tests/basic.ts

```diff
@@ -344,4 +346,660 @@ export const tests: {[name: string] : SSRTest} = {
     ],
     stableSelectors: ['p'],
   },
+
+  'NodePart accepts nested templates': {
```

Comment:
Looks like the first test with 2 adjacent node parts, should that be tested independently?

Reply:
Added separate single & multiple TemplateResult tests.